account/user.go

```diff
@@ -165,3 +165,18 @@ var TestUser = User{
 	Email:    "testdeveloper@testalm.io",
 	Identity: TestIdentity,
 }
+
+// TestIdentity2 only creates in memory obj for testing purposes
+var TestIdentity2 = Identity{
```

Comment:
@hectorj2f I finally opted for a separate PR for this, to keep changes easier to review: https://github.com/almighty/almighty-core/pull/731